### ISSUE TITLE
A small difference between implementation and paper

### ISSUE THREAD

**Issue by kentangSJTU** [yangyanli/PointCNN#68]
Hi Prof. Li,

I ran the code of PointCNN from your group recently, and discovered a small discrepancy between the implementation and your paper. It is mentioned in the paper that in X convolution, both MLP and MLP_delta(in algorithm 1) should not include batch normalization. However, I found in your implementation that batch normalization is actually turned on in all these modules and actually turning on such batch-norms do help me get the results claimed in the paper. So is it not that important to care about whether to have such batch norm layers or not? Thanks for your reply~

Ken

**Comment by yangyanli** [comment]
Hi Ken,

The paper on arXiv is not as updated as the code, where we found with BN is better. We will release a new version of our paper recently.
